public/static/js/version_admin.js

```diff
@@ -194,7 +194,7 @@ mciModule.directive('adminUnscheduleAll', function() {
       '<div class="col-lg-12">' +
         '<div>' +
           'Unschedule all tasks?' +
-            '<div ng-show="version.requester === "merge_test">' +
+            '<div ng-show="version.requester === merge_test">' +
```

Comment:
do we need to adjust the quotes in other occurrences as well?
https://github.com/evergreen-ci/evergreen/pull/2762/files/e3ad51d313675c08b5121d1b30d998abd0ebe81e..ff14df43d6300cf9eed9e0faa7a15fa19cd27b94#diff-ecdb35a4cc1d51de61f1b052ecfb494eR207
https://github.com/evergreen-ci/evergreen/pull/2762/files/e3ad51d313675c08b5121d1b30d998abd0ebe81e..ff14df43d6300cf9eed9e0faa7a15fa19cd27b94#diff-ecdb35a4cc1d51de61f1b052ecfb494eR176
and 
https://github.com/evergreen-ci/evergreen/pull/2762/files/e3ad51d313675c08b5121d1b30d998abd0ebe81e..ff14df43d6300cf9eed9e0faa7a15fa19cd27b94#diff-2c8adb86d56a39580f385577cd287fe2R196

Reply:
I didn't actually mean to remove this quote, sorry

Reply:
does it work with the quote there? (since the first quote is closing the string)

Reply:
Doesn't seem to matter though actually. @john-m-liu how do quotes work which is better

Reply:
The new version in the diff looks right. Javascript matches the specific type of quote